modules/burdastyle_thunder_admin/burdastyle_thunder_admin.module

```diff
@@ -236,3 +236,27 @@ EOT;
 
   return $cssPlaceholder;
 }
+
+
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ */
+function burdastyle_thunder_admin_field_widget_paragraphs_form_alter(&$element, &$form_state, $context) {
+  if ($element['#paragraph_type'] == 'content_reference') {
+    foreach ($element['subform']['field_e_view_mode']['widget']['#options'] as $key => $value) {
+      if (strpos($key, 'content_reference_') !== 0) { // we require valid view modes to
+        // start with content_reference_
+        unset($element['subform']['field_e_view_mode']['widget']['#options'][$key]);
+      }
+    }
+  }
+}
+
+/**
+ * Implements hook_preprocess_views_view().
+ */
+function burdastyle_thunder_admin_preprocess_views_view(&$variables) {
+  if ($variables['view']->id() == 'content_reference_browser') {
```

Comment:
check if $variables['view'] is actually a view. We had similar issues with $variables['node'] being a Drupal\Entity\Node most of the time, but in an edge case it was just a nodeid

Reply:
done